api/query/concrete_query.go

```diff
@@ -117,6 +117,15 @@ func (Link) Size() int { return 1 }
 // Size of Count is the sum of the sizes of its constituent ConcretQuery instances.
 func (c Count) Size() int { return size(c.Args) }
 
+// Size of Is depends on the quality.
```

Comment:
What are these `Size()` methods for, what happens if they return the wrong value?

Reply:
The "size" of the query being performed; not sure exactly when it's used. @mdittmer could answer that.